build.sbt

```diff
@@ -0,0 +1,48 @@
+import sbt.Keys._
+import sbt._
+import Dependencies.{appDependencies, appDependencyOverrides}
+import scoverage.ScoverageKeys
+import uk.gov.hmrc.DefaultBuildSettings.{addTestReportOption, defaultSettings, scalaSettings}
+import uk.gov.hmrc.sbtdistributables.SbtDistributablesPlugin.publishingSettings
+
+val appName = "lightweight-contact-events"
+
+// TODO is it ok to remove this ??
```

Comment:
*Question:* I couldn't see how was `playSettings` and `plugins` variables are used in the first place - so I was a bit hesitant to remove them?... But is it ok to get rid of them?

Reply:
If you know what you are dogin, it's ok to remove.

Reply:
Removed in https://github.com/hmrc/lightweight-contact-events/pull/50/commits/d58b6701549ad8edd7ab19f6aacb495df7984ce4